src/net.cpp

```diff
@@ -383,7 +385,7 @@ CNode* FindNode(const NodeId nodeid)
     return NULL;
 }
 
-CNode* ConnectNode(CAddress addrConnect, const char *pszDest, bool fCountFailure)
+CNode* ConnectNode(CAddress addrConnect, const char *pszDest, bool fCountFailure, bool fFeeler)
```

Comment:
Is it worth the complexity of adding another flag to ConnectNode if all the flag does is alter a log message?
Setting this flag false does not make the connection not a feeler connection.